src/docs/guides/gulp-pipeline/README.md

```diff
@@ -188,6 +188,11 @@ The following example processes all files with a `.js` extension in the `Assets`
 ]
 ```
 
+!!! warning
+    When using wildcards, the order in which files are processed can be nondeterministic due to Gulp's parallelization. This can e.g. cause the concatenated output file to change on a rebuild, without changing the input files. This is a [known behavior of Gulp](https://github.com/gulpjs/gulp/issues/687) but the recommended plugins to fix this are unmaintained.
+
+    If you run into this issue, manually declarate the input files in a static order instead.
```

Comment:
This is why I changed src/OrchardCore.Modules/OrchardCore.Resources/Assets.json. I noticed the same issue randomly happening, but rarely, with Media and TheAdmin too. However, with those, I'm less sure about the scripts being independent of each other (the Trumbowyg plugins are) and thus the order not mattering. What do you think?